lib/utils/report.mjs

```diff
@@ -146,3 +134,83 @@ function printfLike(format, ...args) {
 		return result.replace(/%[ds]/, String(arg));
 	}, format);
 }
+
+/** @typedef {import('stylelint').DisabledRangeObject} DisabledRangeObject */
+/** @typedef {import('stylelint').StylelintPostcssResult} StylelintPostcssResult */
+/** @typedef {import('stylelint').Range} Range */
+/** @typedef {import('stylelint').Problem} Problem */
+
+/**
+ * @param {string} ruleName
+ * @param {number} startLine
+ * @param {DisabledRangeObject} [disabledRanges]
+ */
+function isDisabled(ruleName, startLine, disabledRanges) {
+	if (!disabledRanges) return false;
+
```

Comment:
[suggest] `disabledRanges` seems not undefined: 🤔 

```suggestion
 * @param {DisabledRangeObject} disabledRanges
 */
function isDisabled(ruleName, startLine, disabledRanges) {
```

Reply:
I won't remove `if (!disabledRanges) return false;` just to keep it consistent with the previous code.
We can refactor it later.

Reply:
Make sense.